dentapp/src/components/LoadingSpinner.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+// import Container from 'react-bootstrap/Container'
+import Spinner from 'react-bootstrap/Spinner'
+
+class LoadingSpinner extends Component {
+    state = {
+        isLoading: this.props.handleShow
+    }
+
+    render() {
+
+        return (
+            <>
+                {this.state.isLoading && <Spinner animation="border" role="status">
```

Comment:
use 'Object destructuring'

const {isLoading} = this.state

Reply:
Sure!